common/scala/src/main/scala/whisk/core/entity/WhiskActivation.scala

```diff
@@ -62,7 +62,8 @@ case class WhiskActivation(
     version: SemVer = SemVer(),
     publish: Boolean = false,
     annotations: Parameters = Parameters(),
-    duration: Option[Long] = None)
+    duration: Option[Long] = None,
+    entityType: String)
```

Comment:
bummer! thank you for the improvements to the CLI in that case -- these were some long outstanding features/improvements we hadn't gotten to, I didn't realize you are leaving so soon.